Cargo.toml

```diff
@@ -23,10 +23,11 @@ default = ["avx", "sse", "neon"]
 # On every other platform, these features do nothing, and RustFFT will behave like they are not set.
 #
 # On AArch64, the "neon" feature enables compilation of Neon-accelerated code.
-avx = []
-sse = []
-neon = []
-wasm_simd = []
+std = []
+avx = ["std"]
+sse = ["std"]
+neon = ["std"]
+wasm_simd = ["std"]
```

Comment:
Is there no way to get these on `no_std`?

Reply:
Yeah, my bad. Fixed in [a6f4c2e](https://github.com/ejmahler/RustFFT/pull/124/commits/a6f4c2e1e583d1dc5ed6dd5ed1680814501897bb).

Reply:
Yes, fixed by [b0e38ef](https://github.com/ejmahler/RustFFT/pull/124/commits/b0e38ef6de4b1a4624efb7270ad7335372d5f655), [2017988](https://github.com/ejmahler/RustFFT/pull/124/commits/2017988f53d2e2060989a0ca328976219d56f9c4) and [68d0966](https://github.com/ejmahler/RustFFT/pull/124/commits/68d0966bee60fc789a15522a3d7c5ff7470e9a3b).